src/collections/_documentation/platforms/php/index.md

```diff
@@ -2,43 +2,262 @@
+  $scope->setTag('page_locale', 'de-at');
+});
+```
+
+Several common uses for tags include:
+
+- The hostname of the server
+- The version of your platform (For example, iOS 5.0)
+- The user’s language
+
+Once you’ve started sending tagged data, you’ll see it show up in a few places:
+
+- The filters within the sidebar on the project stream page.
+- Summarized within an event on the sidebar.
+- The tags page on an aggregated event.
+
+We’ll automatically index all tags for an event, as well as the frequency and the last time the Sentry SDK has seen a value. Even more so, we keep track of the number of distinct tags and can assist you in determining hotspots for various issues.
+
+### Setting the Level
+
+You can set the severity of an event to one of five values: 'fatal,' 'error,' 'warning,' 'info,' and 'debug.' 'error' is the default, 'fatal' is the most severe and 'debug' is the least severe.
+
+```php
+Sentry\configureScope(function (Sentry\State\Scope $scope): void {
+  $scope->setLevel(Sentry\Severity::warning());
+});
+```
+
+### Setting the Fingerprint
+
+Sentry uses one or more “fingerprints” to decide how to group errors into issues.
+
+For some very advanced use cases, you can override the Sentry default grouping using the `fingerprint` attribute. In supported SDKs, this attribute can be passed with the event information and should be an array of strings.
+
+If you wish to append information, thus making the grouping slightly less aggressive, you can do that as well by adding the special string `{{default}}` as one of the items.
+
+For more information, see [Aggregate Errors with Custom Fingerprints](https://blog.sentry.io/2018/01/18/setting-up-custom-fingerprints).
+
+## Advanced Usage
+
+### Breadcrumbs
+
+Sentry will automatically record specific events, such as changes to the URL and XHR requests to provide context to an error.
+
+You can manually add breadcrumbs on other events or disable breadcrumbs.
+
+```php
+Sentry\addBreadcrumb(new Sentry\Breadcrumb(
+  Sentry\Breadcrumb::LEVEL_ERROR, 
+  Sentry\Breadcrumb::TYPE_ERROR, 
+  'error_reporting', 
+  'foo bar'
+));
+```
+
+For more information, see:
+
+- [Full documentation on Breadcrumbs]({%- link _documentation/enriching-error-data/breadcrumbs.md -%})
+- [Debug Issues Faster with Breadcrumbs](https://blog.sentry.io/2016/05/04/breadcrumbs)
+
+### Filter Events & Custom Logic
+
+Sentry exposes a beforeSend callback which can be used to filter out information or add additional context to the event object.
+
+```php
+Sentry\init([
+  'dsn' => '___PUBLIC_DSN___',
+  'before_send' => function (Sentry\Event $event): ?Event {
+    return $event;
+  },
+]);
+```
+
+For more information, see:
+
+- [Full documentation on Filtering Events]({%- link _documentation/error-reporting/configuration/filtering.md -%})
+- [Manage Your Flow of Errors Using Inbound Filters](https://blog.sentry.io/2017/11/27/setting-up-inbound-filters)
+
+
+### Integrations
```

Comment:
This seems disconnected here, we either need to list the integrations we have below or move it somewhere else
We have the default integrations on the current docs that can go beneath it or link to them.